sycl/ReleaseNotes.md

```diff
@@ -11,7 +11,7 @@ but can only be used if there are no cross-object dependencies. [f884993d]
 - Added support for per-aspect device code split mode. [9a2c4fe5]
 - Extended support for the large GRF mode to non-ESIMD kernels. [9994934b]
 [ab2a42c6]
-- Implemented the [`sycl_ext_intel_device_architecture`](doc/extensions/supported/sycl_ext_oneapi_device_architecture.asciidoc)
+- Implemented the [`sycl_ext_intel_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc)
```

Comment:
Minor thing: 
sycl_ext_**oneapi**_device_architecture

Reply:
I didn't get what you mean here. Could you elaborate, please?